orders/controller.js

```diff
@@ -0,0 +1,24 @@
+const OrderModel = require('./models/OrderModel');
+
+class OrdersController {
+  static async list({ query }, builder = OrderModel.query()) {
+    return builder
+      .allowEager('[applications]')
+      .eager('applications')
+      .modifyEager('applications', _builder =>
+        _builder.applyFilter('awaiting'))
+      .where(query);
+  }
+  static async create({ eventId, userId, applications }) {
+    return OrderModel
```

Comment:
Adding : ensure the status reflects the current status of the event. It shouldn't be passed by the front-end

Reply:
This should be "approved" or "pending" only, depending on the status of the event ('requiresApproval/not)

Reply:
Sorry. On insert is valid :+1: